src/movies.js

```diff
@@ -1,16 +1,113 @@
+
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let directors = movies.map (function(movie){
+        return movie.director;
+    })
+    return directors;
+} 
+
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
+function howManyMovies(movies){
+
+    let occurance = movies.filter(function(movie) {
```

Comment:
Try to come up with more descriptive variables of what would be inside it like filteredArray or filteredMovies